pkg/service_quotas/service_quotas.go

```diff
@@ -174,14 +178,17 @@ func (s *ServiceQuotas) quotasForService(service string) ([]QuotaUsage, error) {
 // QuotasAndUsage returns a slice of `QuotaUsage` or an error
 func (s *ServiceQuotas) QuotasAndUsage() ([]QuotaUsage, error) {
 	allQuotaUsages := []QuotaUsage{}
-	for _, service := range allServices() {
-		serviceQuotas, err := s.quotasForService(service)
-		if err != nil {
-			return nil, err
-		}
 
-		for _, quota := range serviceQuotas {
-			allQuotaUsages = append(allQuotaUsages, quota)
+	if !s.isAwsChina {
+		for _, service := range allServices() {
+			serviceQuotas, err := s.quotasForService(service)
+			if err != nil {
+				return nil, err
+			}
+
+			for _, quota := range serviceQuotas {
+				allQuotaUsages = append(allQuotaUsages, quota)
+			}
```

Comment:
Would be great if we have some tests for that too :) (I don't believe the tests will pass with those changes)

Reply:
Added